elastic/kibana
# [Fleet] Add new space aware saved object type to Fleet saved object permissions

## Description 

Resolve #192988

Add the new fleet space aware saved object (fleet-agent-policies and fleet-package-policies) to the saved object list accessible with fleet roles.

Before that, those saved object where only accessible by a superuser.

## How to test

With space enabled (feature flag + API call) check a user with fleet access (non super user) can create and access policies
